README.md

```diff
@@ -69,26 +70,61 @@ to some port of your choice e.g.
 "http": [1234, "auto"]
 ```
 
+If you installed OpenSSL in such a way that running `openssl` at a
+command line doesn't work, you also need to adjust the path to the
+OpenSSL binary. This can be done by adding a section to `config.json`
+like:
+
+```
+"ssl": {"openssl": {"binary": "/path/to/openssl"}}
+```
+
 Running Tests Automatically
 ---------------------------
+You will need to install virtualenv before running the cmd line for tests.
+The easiest way to install virtualenv is to add the python scripts directory
+(typically C:\Python27\Scripts) to your `%Path%`
+[Environment Variable](http://www.computerhope.com/issues/ch000549.htm),
+then running easy_install pip
+then running pip install virtualenv
 
-Tests can be run automatically in a browser using the `run` command of
+You will also need to put the appropriate WebDriver executables on the
+machine. For example, inside the .\web-platform-tests\_venv\Scripts directory:
+
+ .\web-platform-tests\_venv\Scripts\chromedriver.exe
+ .\web-platform-tests\_venv\Scripts\MicrosoftWebDriver.exe
+
+ or place them into your `%Path%`
+[Environment Variable](http://www.computerhope.com/issues/ch000549.htm).
```

Comment:
This link is entirely Windows-specific. Ideally we would link to some official documentation rather than a third-party site.